src/Session.cpp

```diff
@@ -21,6 +21,7 @@ std::vector<std::string> Session::extractTags(nlohmann::json& tagList)
 
 void Session::fillContentFromJson(const std::string &configFilePath)
 {
+    content = std::vector<Watchable*>();
```

Comment:
YOU commented that this can be removed because it is already initialized anyway :)

Reply:
auto merge :see_no_evil: 
nice you noticed